gremlin-javascript/src/main/javascript/gremlin-javascript/lib/structure/graph.ts

```diff
@@ -46,43 +44,49 @@ class Graph {
 }
 
 class Element {
-  constructor(id, label) {
-    this.id = id;
-    this.label = label;
-  }
+  constructor(
+    readonly id: string,
+    readonly label: string,
+  ) {}
 
   /**
    * Compares this instance to another and determines if they can be considered as equal.
    * @param {Element} other
    * @returns {boolean}
    */
-  equals(other) {
+  equals(other: Element) {
```

Comment:
other can by `any`

Reply:
Thanks, I've updated the type here to `any`.

Reply:
same for `Path` and `Property`

Reply:
Yes, this make sense, I've updated accordingly.